### PR TITLE
Update helm

### PR DESCRIPTION
**Pull request by suchen-sci**
This pr is mainly used to remove dependencies in our helm charts. So single directory in `helm-charts` can be used to do deployment.

### CODE REVIEW

File: helm-charts/easegress/README.md [hunk 1]
```diff
@@ -58,3 +55,18 @@ helm uninstall easegress -n easegress
 kubectl delete pvc easegress-pv-easegress-0 -n easegress
 # same for easegress-pv-easegress-i...n
 ```
+
+## Parameters
+
+The following table lists the configurable parameters of the MegaEase Easegress.
```
Comment (samutamm) [inline helm-charts/easegress/README.md:61]:
```suggestion
The following table lists the configurable parameters of the MegaEase Easegress Helm installation.
```

File: helm-charts/easegress/README.md [hunk 2]
```diff
@@ -58,3 +55,18 @@ helm uninstall easegress -n easegress
...
+
+| Key | Type | Default | Description |
```
Comment (samutamm) [inline helm-charts/easegress/README.md:63]: This table duplicates the information in `values.yaml` file, forcing to change two places, when something changes here. Given that [values.yaml](https://helm.sh/docs/chart_best_practices/values/) is common to all Helm installations, I would consider removing this table and only referencing values.yaml here, like `Helm parameters and default values are listed in values.yaml`.

File: helm-charts/easegress/templates/PersistentVolume.yaml [hunk 3]
```diff
@@ -25,6 +25,12 @@ spec:
...
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: easegress-storage
+provisioner: kubernetes.io/no-provisioner
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
 
-{{- end }}
+{{ end }}
```
Comment (samutamm) [inline helm-charts/easegress/templates/PersistentVolume.yaml:36]: Nit: `{{- end}}` will not print extra empty line while `{{ end }}` does. Ref: [[controlling whitespace](https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace)]

File: helm-charts/ingress-controller/README.md [hunk 4]
```diff
@@ -54,3 +51,18 @@ helm uninstall ingress-easegress -n ingress-easegress
 kubectl delete pvc easegress-pv-ingress-easegress-0 -n ingress-easegress
 # same for easegress-pv-ingress-easegress-i...n
 ```
+
+## Parameters
```
Comment (samutamm) [inline helm-charts/ingress-controller/README.md:55]: ditto

File: helm-charts/ingress-controller/values.yaml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 service:
   # nodePort for easegress service
-  nodePort: 80
+  nodePort: 30080
```
Comment (samutamm) [inline helm-charts/ingress-controller/values.yaml:3]: I would keep the Ingress port as 80 as it is the value for Kubernetes Ingress and [HTTP traffic in general](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers).

File: helm-charts/easegress/README.md (hunk 2 above)
Comment (suchen-sci) [reply helm-charts/easegress/README.md:63]: in kubesphere, https://github.com/kubesphere/helm-charts/tree/master/src/ every readme has parameter part to show what the parameter are...

File: helm-charts/ingress-controller/values.yaml (hunk 5 above)
Comment (suchen-sci) [reply helm-charts/ingress-controller/values.yaml:3]: this is nodePort... the real port out k8s to access is defined by `Service`. nodePort is fine to be any number...

File: helm-charts/easegress/README.md (hunk 2 above)
Comment (suchen-sci) [reply helm-charts/easegress/README.md:63]: So i will keep the parameters...

**Review by samutamm — APPROVED** [review]

**Review by localvar — APPROVED** [review]

**Pull request merged**